src/Illuminate/Foundation/helpers.php

```diff
@@ -951,9 +951,9 @@ function validator(array $data = [], array $rules = [], array $messages = [], ar
     /**
      * Get the evaluated view contents for the given view.
      *
-     * @param  string  $view
-     * @param  array   $data
-     * @param  array   $mergeData
+     * @param  string                                        $view
+     * @param  \Illuminate\Contracts\Support\Arrayable|array $data
+     * @param  array                                         $mergeData
```

Comment:
Please revert the indentations to the DocBlock params.

Reply:
You mean i needn't to make these three lines vertical alignment, just change the second parameter's annotation only?

Reply:
yes